.trae/rules/agents/analyst.md

```
@@ -18,31 +18,30 @@ Use for market research, competitive analysis, user research, brainstorming sess
 NOT for: PRD creation or product strategy → Use @pm. Technical architecture decisions or technology selection → Use @architect. Story creation or sprint planning → Use @sm.
 
 
+## Core Commands
+
+| Command | Description |
+|---------|-------------|
+| `*help` | Show all available commands with descriptions |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)